module/VuFind/tests/integration-tests/src/VuFindTest/Mink/LoggingTest.php

```diff
@@ -431,7 +465,6 @@ public function testNoEmailLoggingWhenDisabled(): void
         $emailLogPath = $this->getEmailLogPath();
         if (file_exists($emailLogPath)) {
             $loggedEmails = trim(file_get_contents($emailLogPath));
-            print_r($loggedEmails);
```

Comment:
I'm puzzled by this phantom line -- it shows in the PR diffs but does not exist in the file.